waku/v2/protocol/waku_rln_relay/utils.nim

```diff
@@ -14,11 +14,13 @@ import
   libp2p/protocols/pubsub/rpc/messages,
   libp2p/protocols/pubsub/pubsub,
   stew/results,
-  stew/[byteutils, arrayops, endians2],
+  stew/[byteutils, arrayops, endians2]
+import
   rln, 
-  waku_rln_relay_constants,
-  waku_rln_relay_types,
-  waku_rln_relay_metrics,
+  constants,
+  types,
+  protocol_metrics
+import
```

Comment:
[_use explicit import paths (e.g. ./ for modules in same path_)
](https://hackmd.io/5oIVOuQcSM2WKrnDLUPcpg?view#:~:text=use%20explicit%20import%20paths%20(e.g.%20./%20for%20modules%20in%20same%20path))

Reply:
Addressed in e9d15c53